lastpass-cli/plan.sh

```diff
@@ -0,0 +1,50 @@
+pkg_name=lastpass-cli
+pkg_origin=core
+pkg_version=1.3.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
```

Comment:
I think this package is GPL
https://github.com/lastpass/lastpass-cli/blob/master/COPYING

Reply:
Yes, looks like that's so, oops.  Searching in contributing it actually says `GPL-2+`.  I will change it to that.